ui/partials/perf/dashboard.html

```diff
@@ -1,37 +0,0 @@
-<div class="container-fluid">
-  <div class="alert alert-danger" ng-show="revisionNotFound" role="alert">
-    Couldn't find revision {{revision}} for {{selectedRepo.name}}
-  </div>
-  <div ng-if="!revision">
-    <h1>{{header}}</h1>
-    <p>{{descP1}}</p>
-    <p>{{descP2}} <a href="{{linkUrl}}">{{linkDesc}}</a></p>
-    <div class="form-group">
-      Branch to compare:
-      <select ng-change="globalOptionsChanged(selectedRepo, selectedTimeRange)"
-              ng-model="selectedRepo"
-              ng-options="repo.name for repo in projects">
-      </select>
-      Time range to sample:
-      <select ng-model="selectedTimeRange"
-              ng-options="timeRange.text for timeRange in timeRanges track by timeRange.value"
-              ng-change="globalOptionsChanged(selectedRepo, selectedTimeRange)">
-      </select>
-    </div>
-  </div>
-  <div ng-if="dataLoading">
-    Loading all results, please wait a minute...
-    <img src="../../img/dancing_cat.gif" />
-  </div>
-  <div ng-if="!dataLoading && !revisionNotFound">
-    <ph-compare-table
-       base-title={{baseTitle}}
-       new-title={{variantTitle}}
-       titles="titles"
-       test-list="testList"
-       compare-results="compareResults"
-       filter-options="filterOptions"
-       release-blocker-criteria="1">
```

Comment:
I believe `release-blocker-criteria` corresponds to the `releaseBlockerCriteria` references in some of the remaining files (I hate Angular's automatic naming conversion) - which means they are now unused after this template is deleted. As such I think anything to do with "blockers" (eg `releaseBlockerCriteria`, `showOnlyBlockers`, `isBlocker`) can now be deleted from `compare.js`, the compare templates and a few other places.

Reply:
This review comment was not addressed?

Reply:
Er, addressed below :)